producer/smpolicy.go

```diff
@@ -340,17 +354,25 @@ func HandleUpdateSmPolicyContextRequest(request *httpwrapper.Request) *httpwrapp
 	logger.SMpolicylog.Infoln("handle UpdateSmPolicyContext")
 	requestDataType := request.Body.(models.SmPolicyUpdateContextData)
 	smPolicyID := request.Params["smPolicyId"]
+	getResponse, getProblemDetails := getSmPolicyContextProcedure(smPolicyID)
+	smPolicyDnn := "UNKNOWN_DNN"
+	if getProblemDetails != nil {
```

Comment:
1. getResponse would be nil if getProblemDetails is not nil.
2. Can we have consistency in naming these variables?